File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Service/src/Exceptions/ContestDataException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Ubb.BikeContest.Services.src.Exceptions;

public class ContestDataException : Exception
{
    public ContestDataException() : base() { }

    public ContestDataException(string msg) : base(msg) { }

    public ContestDataException(string msg, Exception ex) : base(msg, ex) { }

}

File: Anul II/Medii de proiectare si programare/Laborator/lab11/java/services/src/main/java/com/ubb/exceptions/ContestDataException.java
package com.ubb.exceptions;

public class ContestDataException extends Exception {
    public ContestDataException() {
    }

    public ContestDataException(String message) {
        super(message);
    }

    public ContestDataException(String message, Throwable cause) {
        super(message, cause);
    }
}
